tests/io_adapters_test.py

```diff
@@ -2,18 +2,25 @@
 from tensorflow import keras
 import numpy as np
 import unittest
-
-from lib import io_adapter as adp
-
-
+from lib import io_adapter as  adp
 
 class InputAdapterTest(tf.test.TestCase):
+  """Input adapter test class.
+  Methods:
+  test_output_shape: tests if the output of the adapter is of correct
+  shape.
+  test_small_in_shape: checks if an error is raised when the input shape
+  is too small.
+  test_non_divisible: checks if an error is raised when input height or 
+  width is not devisible by `size`.
+  _create_default_adapter: creates an adapter with default parameters given
+  an input shape."""
   def setUp(self):
     super(InputAdapterTest, self).setUp()
     self.default_size = 32
     self.default_depth = 64
 
-  # tests if the output of the adaptes is of correct shape
+  # tests if the output of the adapter is of correct shape
   def test_output_shape(self):
```

Comment:
This should become a function docstring:

```suggestion
  def test_output_shape(self):
    """Tests if the output of the adapter is of correct shape."""
```

Same docstrings should be added to other functions (not necessarily all) by moving them from the above class docstring.